soc/st/stm32/stm32u0x/soc.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief System/hardware module for STM32U0 processor
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <stm32_ll_bus.h>
+#include <stm32_ll_pwr.h>
```

Comment:
Is this header used?

Reply:
For the moment no, i removed the header file